client/src/lifecycleManager.js

```diff
@@ -162,11 +162,9 @@ class LifecycleManager extends LuigiClientBase {
         .find(cookie => cookie == 'luigiCookie=true');
     }
     if (luigiCookie === 'luigiCookie=true') {
-      window.parent.postMessage({ msg: 'luigi.third-party-cookie', tpc }, '*');
       document.cookie = luigiCookieKey + '=; Max-Age=-99999999; SameSite=None; Secure';
     } else {
       tpc = 'disabled';
-      window.parent.postMessage({ msg: 'luigi.third-party-cookie', tpc }, '*');
       console.warn('Third party cookies are not supported!');
     }
   }
```

Comment:
This whole function here, seems like it was created on this PR
https://github.com/SAP/luigi/pull/970/files long time ago. 
I am guessing that since we remove third party cookie check functionality, we undo all that was done in that PR.
so then remove all the _tpcCheck function and its usage altogether. 

That would make sense, but lets also discuss it in daily further. (ex: Should we remove it altogether/Should we keep a console warning anyway that third party cookies are not supported, or do we let Chrome log its own warnings )